docs/user/logs.md

```diff
@@ -0,0 +1,597 @@
+  "observedTime": "2022-05-23T15:04:52.200000000",
+  "attributes": {
+   "log.file.path": "/var/log/pods/myNamespace_myPod-<containerID>/myContainer/<containerRestarts>.log",
+   "log.iostream": "stdout",
+   "log.original": "{\"level\": \"warn\",\"message\": \"This is the actual message\",\"tenant\": \"myTenant\",\"trace_id\": \"123\"}",
+   "tenant": "myTenant",
+  },
+  "resourceAttributes": {
+    "k8s.container.name": "myContainer",
+    "k8s.container.restart_count": "<containerRestarts>",
+    "k8s.pod.name": "myPod",
+    "k8s.namespace.name": "myNamespace"
+  },
+  "body": "This is the actual message",
+  "severityNumber": 13,
+  "severityTex": "warn",
+  "trace_id": 123
+}
+```
+
+### 5. Deactivate OTLP Logs
+
+By default, `otlp` input is enabled.
+
+To drop the push-based OTLP logs that are received by the Log gateway, define a LogPipeline that has the `otlp` section disabled as an input:
+
+```yaml
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: LogPipeline
+metadata:
+  name: backend
+spec:
+  input:
+    application:
+      enabled: true
+    otlp:
+      disabled: true
+  output:
+    otlp:
+      endpoint:
+        value: https://backend.example.com:4317
+```
+
+With this, the agent starts collecting all container logs, while the push-based OTLP logs are dropped by the gateway.
+
+### 6. Deploy the Pipeline
+
+To activate the LogPipeline, apply the `logpipeline.yaml` resource file in your cluster:
+
+```bash
+kubectl apply -f logpipeline.yaml
+```
+
+### Result
+
+You activated a LogPipeline and logs start streaming to your backend.
+
+To check that the pipeline is running, wait until the status conditions of the LogPipeline in your cluster have status `True`:
+
+```bash
+kubectl get logpipeline
+NAME      CONFIGURATION GENERATED   GATEWAY HEALTHY   AGENT HEALTHY   FLOW HEALTHY
+backend   True                      True              True            True        
+```
+
+## Kyma Modules With Logging Capabilities
+
+Kyma bundles modules that can be involved in user flows. If you want to collect all logs of all modules, enable the `application` input for the `kyma-system` namespace.
+
+### Istio
+
+The Istio module is crucial as it provides the [Ingress Gateway](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/). Typically, this is where external requests enter the cluster scope. Furthermore, every component that’s part of the Istio Service Mesh runs an Istio proxy. Using the Istio telemetry API you can enable access logs for the Ingress Gateway and the proxies individually.
+
+The Istio module is configured with an [extension provider](https://istio.io/latest/docs/tasks/observability/telemetry/) called `kyma-logs`. To activate the provider on the global mesh level using the Istio [Telemetry API](https://istio.io/latest/docs/reference/config/telemetry), place a resource to the `istio-system` namespace. The following code samples help setting up the Istio logging feature:
+
```

Comment:
Line 508 ends with "The following code samples..." and line 510 starts with "The following example...". Atm, this sounds a bit redundant, but I assume you'll keep adding more examples so that the sentence in line 508 is justified.

Reply:
done